lms/djangoapps/course_api/blocks/transformers/tests/test_milestones.py

```diff
@@ -19,7 +20,7 @@
 @attr('shard_3')
 @ddt.ddt
 @patch.dict('django.conf.settings.FEATURES', {'ENABLE_SPECIAL_EXAMS': True, 'MILESTONES_APP': True})
-class MilestonesTransformerTestCase(CourseStructureTestCase, MilestonesTestCaseMixin):
+class MilestonesTransformerTestCase(CourseStructureTestCase, MilestonesTestCaseMixin, ModuleStoreTestCase):
```

Comment:
The problem I've seen in the past with inheriting from multiple `TestCase` classes is that the `TestCase` methods don't call `super` and so the methods in the subsequently inherited `TestCase` class don't actually get called.